sdk/cosmosdb/ci.yml

```diff
@@ -0,0 +1,44 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/cosmosdb/
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+  paths:
+    include:
+      - sdk/cosmosdb/
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+    parameters:
+      # Unit-tests are being skipped. So, matrix is not needed.
+      Matrix: {}
```

Comment:
Upon further thinking it makes more sense to use a conditional to exclude the `UnitTest` steps with some parameter in `archetype-sdk-client.yml` like `RunUnitTest` which defaults to `true`. 

The fact that the template doesn't have a matrix parameter suggests to me that we didn't want to give people the option of altering the matrix. @mitchdenny what do you think? 

PS. The idea of messing with the matrix was mine originally with the goal of reducing the amount of agents we tie up running tests that amount to "skipped everything"

Reply:
Our model does allow for some flexibility around execution for special scenarios. But every time we deviate from the norm we make maintenance across the repo just a little bit harder. On a case by case basis it doesn't seem like such a big deal but it'll be death by a million cuts.

What is the reasoning behind disabling the unit tests? It feel like we are leaving ourselves light on coverage here.

Reply:
@southpolesteve -- 100% we will be running tests on each check in and we will not be stepping backwards. This discussion centers around the finer points of how we fit testing cosmos into our broader architecture.

In this case, the `UnitTest` steps are a no-op (https://dev.azure.com/azure-sdk/public/_build/results?buildId=98556&view=logs&j=1c778e8e-596b-5a0a-22c2-460364725d7c) and the integration tests are where all of the work is happening. If the unit tests didn't result in "skip" we'd run them against at least the default matrix.

CI testing resources constrained so we want to keep the amount of VMs that we spin up for the purpose of saying "skipped" to a minimum to keep our test pool contention down.

Reply:
@mitchdenny and I just spoke. We can merge this now and I'll refactor to better support unified pipelines